IOS new thread exception crash report fix (#453)
Microsoft/AppCenter-SDK-Unity: CHANGELOG.md

# App Center SDK for Unity Change Log

## Release 3.2.1 (in development)

### App Center

#### iOS

> * **[Fix]** Fix new thread exception crash report don't send.

Reviewer: ```suggestion
* **[Fix]** Fix reporting crashes caused by a thread exception.
```
